lib/src/fixed.dart

```diff
@@ -423,25 +427,20 @@ class Fixed implements Comparable<Fixed> {
   /// Formats a [Fixed] value into a String according to the
   /// passed [pattern].
   ///
-  /// If [invertSeparator] is true then the role of the '.' and ',' are
-  /// reversed. By default the '.' is used as the decimal separator
-  /// whilst the ',' is used as the grouping separator.
+  /// [decimalSeparator] and [groupSeparator] are optional and
+  /// default to '.' and ',' respectively.
   ///
   /// 0 A single digit
   /// # A single digit, omitted if the value is zero
-  /// . or , Decimal separator dependant on [invertSeparator]
```

Comment:
these comments need to remain but  be revised as . , and - are still valid chars in the pattern.

Reply:
oh and the space character as well.